src/constructs/EcsServiceFactory.ts

```diff
@@ -251,13 +266,21 @@ export class EcsServiceFactory {
     integration.node.addDependency(service);
     integration.node.addDependency(this.props.link);
 
-    const route = new CfnRoute(this.scope, `${id}-route`, {
-      apiId: this.props.api.apiId,
-      routeKey: `ANY /${path}/{proxy+}`,
-      target: `integrations/${integration.ref}`,
-    });
+    let route;
+    if (isRootPath) {
```

Comment:
Even goed kijken of dit allemaal blijft werken? Ik weet niet precies wat jullie hier hebben gedaan om gzac aan de praat te krijgen?